Wise_Owl_Library/Repositories/WiseOwlLibraryDbContext.cs

```diff
@@ -13,10 +14,18 @@ public class WiseOwlLibraryDbContext(DbContextOptions<WiseOwlLibraryDbContext> o
         protected override void OnModelCreating(ModelBuilder modelBuilder)
         {
             modelBuilder.Entity<BookEntity>()
-                .HasMany(b => b.Authors)
-                .WithMany(a => a.Books);
+                .HasMany(b => b.Authors);
+
+            modelBuilder.Entity<AuthorEntity>()
+                .HasMany(a => a.Books);
```

Comment:
Tohle je zvláštní vazba, teď to tam bude vypadat že to je propojené, ale ve skutečnosti tam v db vazba reálně nebude ne? Nebo si s tím EF poradí? Docela by mě zajímalo jak budou vypadat constrains a schéma. a jak se to zachová např při smazání pouze autora, nebo pouze knihy.